macaroonbakery/third_party.py

```diff
@@ -16,46 +16,25 @@ def legacy_namespace():
 class ThirdPartyCaveatInfo(namedtuple(
     'ThirdPartyCaveatInfo',
     'condition, first_party_public_key, third_party_key_pair, root_key, '
-        'caveat, version, ns')):
+        'caveat, version, namespace')):
     '''ThirdPartyCaveatInfo holds the information decoded from
     a third party caveat id.
+    param condition holds the third party condition to be discharged.
```

Comment:
Whitespace between parameters?
Also, either use ":param" or capitalize the sentences?